tensorrt_llm/bench/benchmark/low_latency.py

```diff
@@ -289,6 +283,17 @@ def latency_command(
     kwargs = kwargs | runtime_config.get_llm_args()
     kwargs['backend'] = options.backend
 
+    # Set environment variables for setting runtime options.
```

Comment:
Is it better to move these to llm_args.py to make the common users get the same default envs as trtllm-bench?